src/Tags/Range.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Statamic\Tags;
+
+use Statamic\Tags\Tags;
+
+class Range extends Tags
+{
+    public function index()
+    {
+        $from   = $this->params->int('from', 1);
+        $to     = $this->params->int('to', null);
+        $times  = $this->params->int('times', null);
```

Comment:
This can be changed to `$this->params->int(['to', 'times']);`
Passing an array will let you get the value from any matching parameters.

Then you can remove the `if ($times)` stuff below.